packages/combobox/src/index.tsx

```diff
@@ -710,13 +747,15 @@ export const ComboboxOption: ComponentWithForwardedRef<
   "li",
```

Comment:
I believe the build is failing because `ComponentWithForwardedRef` does not work correctly. You can test it by writing the following code.

`type test = ComponentWithForwardedRef<"div", { something: string }>`

TypeScript thinks the type is `any` although when I look in the `dist` folder of my project, it seems to be working fine. 🤔 

![image](https://user-images.githubusercontent.com/28842311/76692893-4e4ecc00-665d-11ea-88ef-7c03b7b176bb.png)